cmd/mdmctl/apply.go

```diff
@@ -68,9 +78,90 @@ Examples:
 
 func (cmd *applyCommand) applyBlueprint(args []string) error {
 	flagset := flag.NewFlagSet("blueprints", flag.ExitOnError)
+	var (
+		flBlueprintPath    = flagset.String("f", "", "filename of blueprint JSON to apply")
+		flNewBlueprintPath = flagset.String("n", "", "filename of new blueprint JSON to create")
```

Comment:
I think only the -f flag should exist. 
The second one is confusing. Is it the template?